app/views/embeddable/multiple_choices/_form.html.haml

```diff
@@ -39,6 +53,24 @@
     :javascript
       $(function() {
           $('.wysiwyg').wysiwyg({"controls": { "html": { "visible": true }, "h1": { "visible": false }, "h2": { "visible": false }, "h3": { "visible": false } } });
+
+          var $radios = $('#embeddable_layout_options input'),
+              show_as_menu = $('##{show_as_menu_id}');
+
+          function toggle_$radios(element) {
+            if($(element).is(':checked')){
+              $radios.attr('disabled',true);
+            }else{
+              $radios.removeAttr('disabled');
+            }
+          }
+
+          show_as_menu.click(function() {
+            toggle_$radios(this);
+          });
+
+          toggle_$radios(show_as_menu);
```

Comment:
Sorry for nit picking, but for consistency:

1. In Javascript we use camel case so the variable should actually be `showAsMenu`.
2. I used the `$` in front of the `radios` variable because it is jquery object.  If you are going to do that, then you should put the `$` in front of the `showAsMenu` variable as well.
3. It is not common to put a `$` in the middle of a function so the function should be `toggleRadios`
4. Since you have a reference to `showAsMenu` already, if you use that directly in `toggleRadios`, Then you can just pass `toggleRadios` directly to the click function: `showAsMenu.click(toggleRadios)` and you can call it directly below: `toggleRadios()`.
5. Finally the name `toggleRadios` doesn't seem right because that implies that the function will just flip the radio buttons  regardless of the value of showAsMenu. A better name would be `updateRadios`.